src/main/scala/nl.knaw.dans.easy.deposit/servlets/package.scala

```diff
@@ -37,14 +37,17 @@ package object servlets extends DebugEnhancedLogging {
   private val post = "-compress(ed)?"
   private val contentTypeZipPattern = s"application/(($pre($post)?)|(x$post))"
 
+  val contentTypeJson: (String, String) = "content-type" -> "application/json;charset=UTF-8"
```

Comment:
although this 'hardcoded' way seems fine, I think there is a method in Scalatra that does this. Something like `contentType = "..."`. Have you tried using that one? Or did you explicitly choose for this way?

Reply:
Current usage of this hard-coded constant:
https://github.com/DANS-KNAW/easy-deposit-api/blob/ef76d3475e8bb8c1b9f46453abce6addffdf90a7/src/main/scala/nl.knaw.dans.easy.deposit/servlets/DepositServlet.scala#L75

Scalatra's convenience field is not convenient at all in this code pattern of this specific servlet. See the drawbacks of setting it with `before` in rejected PR #92